app/views/layouts/_pagination.html.erb

```diff
@@ -0,0 +1,4 @@
+<nav class="pagination" role="navigation" aria-label="pagination">
+  <a href="<%= repositories_path(:page => @pagination[:prev]) %>" class = "pagination-previous">Previous</a>
+  <a href="<%= repositories_path(:page => @pagination[:next]) %>" class = "pagination-next">Next</a>
+</nav>
```

Comment:
Enable the 'add newline at end of file'  option in your editor. That will help avoid this sort of problems